parsec/backend/app.py

```diff
@@ -124,6 +124,11 @@ async def handle_client_websocket(self, stream, event, first_request_data=None):
             client_ctx, error_infos = await do_handshake(self, transport)
             if not client_ctx:
                 # Invalid handshake
+                if error_infos.get("reason", "") == "Expired organization":
```

Comment:
I was filling that is pretty weak to do it like that, and i am not sure what can be the good approach for it. Maybe `do_handshake` should raise some excpetions when the handshake fails. 

I'll create an issue about it because that looks like a bit outside of this scope

Reply:
:+1: